src/Sentry/Internal/AotHelper.cs

```diff
@@ -13,10 +13,31 @@ static AotHelper()
         IsTrimmed = CheckIsTrimmed();
     }
 
+
     [UnconditionalSuppressMessage("Trimming", "IL2026: RequiresUnreferencedCode", Justification = AvoidAtRuntime)]
     private static bool CheckIsTrimmed()
     {
+        if (Check("publishtrimmed"))
+            return true;
+
+        if (Check("publishaot"))
+            return true;
```

Comment:
What if `Check` does find a build property and is able to parse it but the value is `false` (not `true`)? Wouldn't this logic ignore that value and proceed to the fallback logic (which really only detects whether publishTrimmed is true)?

Reply:
Agreed.  I will make this update

Reply:
Done